parquet-format-structures/src/main/java/org/apache/parquet/format/Util.java

```diff
@@ -198,8 +301,16 @@ public void consume(RowGroup rowGroup) {
           }
         })));
       }
+      
+      InputStream from;
+      if (null == decryptor) {
+        from = input;
+      }
+      else {
```

Comment:
nit: please move this line to the end of last line.

Reply:
nit, but not yet addressed, @ggershinsky could you please apply Junjie's recommendation?

Reply:
will do, sorry for missing this.